### PR TITLE
Remove duplicate, identical tests

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

File: clients/client/src/test/java/org/projectnessie/client/TestNessieHttpClient.java [hunk 1]
```diff
@@ -45,36 +41,6 @@ static void setupTracer() {
     JaegerTestTracer.register();
```
Comment (nastra) [inline clients/client/src/test/java/org/projectnessie/client/TestNessieHttpClient.java:41]: I think 
```
@BeforeAll
  static void setupTracer() {
    JaegerTestTracer.register();
  }
```
can be removed now right?

File: clients/client/src/test/java/org/projectnessie/client/http/TestHttpClientBuilder.java [hunk 2]
```diff
@@ -145,49 +141,6 @@ void testAuthBasic(Function<HttpClientBuilder, HttpClientBuilder> config) throws
                     UTF_8));
   }
 
-  @Test
-  void testTracing() throws Exception {
```
Comment (nastra) [inline clients/client/src/test/java/org/projectnessie/client/http/TestHttpClientBuilder.java:149]: where are we testing this now, given that we're removing it here?

Comment (snazy) [reply clients/client/src/test/java/org/projectnessie/client/http/TestHttpClientBuilder.java:149]: In the class you mentioned in the above comment

File: clients/client/src/test/java/org/projectnessie/client/TestNessieHttpClient.java (hunk 1 above)
Comment (snazy) [reply clients/client/src/test/java/org/projectnessie/client/TestNessieHttpClient.java:41]: Ehm - no - `testTracing` is in this class

Comment (nastra) [reply clients/client/src/test/java/org/projectnessie/client/TestNessieHttpClient.java:41]: then I guess we can remove ^ from `TestHttpClientBuilder` :)

File: clients/client/src/test/java/org/projectnessie/client/http/TestHttpClientBuilder.java (hunk 2 above)
Comment (nastra) [reply clients/client/src/test/java/org/projectnessie/client/http/TestHttpClientBuilder.java:149]: yeah I got confused by what is where, so def. makes sense to clean this up here

**Pull request merged**